pipelines/infrastructure/grizzly/grizzly.yaml

```diff
@@ -0,0 +1,130 @@
+---
+resources:
+- name: build-schedule
+  type: time
+  icon: clock
+  source:
+    interval: 1h
+- name: gitrepo
+  type: git
+  icon: github
+  source:
+    branch: main
+    uri: git@github.com:mitodl/grafana-dashboards.git
+    private_key: ((grafana-dashboards-private-key))
+jobs:
+- name: commit-managed-dashboards-from-ci
+  build_log_retention:
+    days: 2
+  plan:
+  - get: build-schedule
+    trigger: true
+  - get: gitrepo
+    trigger: false
+  - task: get-managed-dashboards-from-ci
+    config:
+      inputs:
+      - name: gitrepo
+      outputs:
+      - name: changed-repo
+      platform: linux
+      image_resource:
+        type: registry-image
+        source:
+          repository: grafana/grizzly
+          tag: 0.2.0-beta3-amd64
+      run:
+        path: sh
+        args:
+        - -exc
+        - |
+          export GRAFANA_TOKEN="((grafana-ci-token))"
```

Comment:
You should be able to create a JSON object under a `grizzly` path and use `((grafana.ci_token))`, `((grafana.ci_url))`, etc.